tests/test_api.py

```diff
@@ -44,92 +45,34 @@ def configs(mocked_plugin_loader: type[PluginLoader]):  # noqa: F811
     return list(PluginLoader.get_supported_configs().values())
 
 
-def get_combinations(
-    provider_cfgs: list[ProviderConfig],
-) -> Generator[VariantDescription]:
-    """Generate all possible combinations of `VariantProperty` given a list of
-    `ProviderConfig`. This function respects ordering and priority provided."""
+def test_get_variant_hashes_by_priority_roundtrip(mocker, configs):
+    """Test that we can round-trip all combinations via variants.json and get the same
+    result."""
 
-    assert isinstance(provider_cfgs, (list, tuple))
-    assert len(provider_cfgs) > 0
-    assert all(isinstance(config, ProviderConfig) for config in provider_cfgs)
+    def get_or_skip_combinations() -> Generator[VariantDescription]:
```

Comment:
Why do you need that? The test data is static, so the combination count is static too, so you're either never or always skipping the test.

Reply:
True. Removing